arch/risc-v/src/mpfs/mpfs_ihc.c

```diff
@@ -352,7 +349,7 @@ static uint32_t mpfs_ihc_context_to_remote_hart_id(ihc_channel_t channel)
         }
       else
         {
-          DEBUGPANIC();
```

Comment:
I don't know how the driver is supposed to work, but one example here is to change the return value to int / int32 and return -1 (which is for sure an invalid hart id), instead of crashing here.

Reply:
Right, sounds good, will remove PANICs